src/main/resources/view/NormalMainPanel.fxml

```diff
@@ -11,7 +11,7 @@
 
     <TabPane fx:id="tabPane"
              tabClosingPolicy="UNAVAILABLE">
-        <!-- Students and rooms tabs will be created programatically -->
+        <!-- Students and rooms tabs will be created programmatically -->
```

Comment:
🤣

Reply:
What can I say, IntelliJ suggested it :woman_shrugging: